### DISCUSSION TITLE
MinIO  TLS certificate change name

### DISCUSSION THREAD

**Discussion by leejamein** [minio/minio#19324] [Q&A] [answered]
In MinIO DOCS and the MinIO I currently configure, the TLS certificate names are private.key and public.crt. Can I use certificates with names other than the above?

![image](https://github.com/minio/minio/assets/87750537/cbee2db3-3faa-4ca1-85e4-a660e91ae0d5)

![image](https://github.com/minio/minio/assets/87750537/d102d0a5-abe4-41f7-b7f8-b6deafa7bc07)

**Comment by harshavardhana** [comment, marked as answer]
No

**Reply by leejamein** [reply]
Thank you for answer
